frontend/src/app/projects/dashboard/metrics/[projectKey]/page.tsx

```diff
@@ -0,0 +1,153 @@
+    <div className="flex flex-col gap-4">
+      {metrics && (
+        <>
+          <div className="flex items-center justify-between">
+            <h1 className="text-2xl font-bold">{metrics.projectName}</h1>
+            <MetricsScoreCircle score={metrics.score} />
+          </div>
+          <div className="grid grid-cols-1 gap-4 md:grid-cols-2">
+            <GeneralCompliantComponent
+              title="Funding Requirements Compliant"
+              icon={faDollar}
+              compliant={metrics.isFundingRequirementsCompliant}
+            />
+            <GeneralCompliantComponent
+              title="Leader Requirements Compliant"
+              icon={faHandshake}
+              compliant={metrics.isLeaderRequirementsCompliant}
+            />
+          </div>
+          <div className="grid grid-cols-1 gap-4 md:grid-cols-3">
+            <DashboardCard title="Stars" icon={faStar} stats={millify(metrics.starsCount)} />
+            <DashboardCard title="Forks" icon={faCodeFork} stats={millify(metrics.forksCount)} />
+            <DashboardCard
+              title="Contributors"
+              icon={faPeopleGroup}
+              stats={millify(metrics.contributorsCount)}
+            />
+          </div>
+          <div className="grid grid-cols-1 gap-4 md:grid-cols-3 lg:grid-cols-4">
+            <DashboardCard
+              title="Open Issues"
+              icon={faExclamationCircle}
+              stats={millify(metrics.openIssuesCount)}
+            />
+            <DashboardCard
+              title="Total Issues"
+              icon={faFolderOpen}
+              stats={millify(metrics.totalIssuesCount)}
+            />
+            <DashboardCard
+              title="Unassigned Issues"
+              icon={faTag}
+              stats={millify(metrics.unassignedIssuesCount)}
+            />
+            <DashboardCard
+              title="Unanswered Issues"
+              icon={faQuestionCircle}
+              stats={millify(metrics.unansweredIssuesCount)}
+            />
+          </div>
+          <div className="grid grid-cols-1 gap-4 md:grid-cols-3">
+            <DashboardCard
+              title="Open Pull Requests"
+              icon={faCodePullRequest}
+              stats={millify(metrics.openPullRequestsCount)}
+            />
+            <DashboardCard
+              title="Recent Releases"
+              icon={faRocket}
+              stats={millify(metrics.recentReleasesCount)}
+            />
+            <DashboardCard
+              title="Total Releases"
+              icon={faTags}
+              stats={millify(metrics.totalReleasesCount)}
+            />
+          </div>
+          <BarChart
+            title="Days Metrics"
+            icon={faChartArea}
+            labels={[
+              'Project Age',
+              'Days Since Last Commit',
+              'Days Since Last Release',
+              'Days Since Last Pull Request',
+              'Days Since OWASP Page Last Update',
```

Comment:
Can we add a tooltip to this to show the full text when it gets truncated? 
<img width="425" height="271" alt="Screenshot 2025-07-18 at 5 16 41 PM" src="https://github.com/user-attachments/assets/7da238e4-6be4-4338-8b10-931ba2ea1c85" />

Reply:
@ahmedxgouda Gotcha! I'll check that too! Thank you!